blog/nse_portfolio_sorts/index.qmd

```diff
@@ -0,0 +1,931 @@
+
+This blog article will teach you how to do portfolio sorts with non-standard errors in mind, i.e., vary over possible decisions to gain a deeper insight into return differentials. We will estimate one premium's distribution instead of a single return differential. Tidy Finance with R is the inspiration for the code alongside the replication code for WWW in this [Github repository](https://github.com/patrick-weiss/PortfolioSorts_NSE). Ultimately, you will know how to sort portfolios on the variable *asset growth* in nearly 70,000 ways.
+
+This blog post is long, albeit not overly complicated. If you are new to `R` or portfolio sorts, I suggest reading the Chapter [Size sorts and p-hacking](https://www.tidy-finance.org/size-sorts-and-p-hacking.html) before jumping into this post. Unfortunately, I have to skip many interesting points on the implementation and the economic background (read WWW for this part). Otherwise, this blog would be far too long. If there is a need to expand on individual aspects, let me know. 
+
+# Data
+
+First, we need some data. On the one hand, we need the monthly return time series for the CRSP universe. On the other hand, we need some accounting data from Compustat for constructing the sorting variable itself. To save space and because there is a chapter in Tidy Finance on it, I refer you to our chapter on [WRDS, CRSP, and Compustat](https://www.tidy-finance.org/wrds-crsp-compustat.html) for the details downloading the data. Here, we only read the data from my preprepared SQLite database.
+
+We first need a few packages. The `tidyverse` (of course) and the `RSQLite`-package for the database. Additionally, we connect to my database, which contains all the necessary data. The prefix `../` in the path argument moves one directory up.
+
+```{r}
+#| warning: false
+library(tidyverse)
+library(RSQLite)
+
+# Database
+data_tidy_nse <- dbConnect(SQLite(), 
+                           "../../data/data_nse.sqlite", 
+                           extended_types = TRUE)
+```
+
+Next, I load the necessary stock market (CRSP) and accounting (Compustat) data from my SQLite database.
+
+```{r}
```

Comment:
Ich versteh nicht, warum der Code Chunk da ist. Ist das nicht einfach nur eine Kopie von unserem Chapter? Sind die Daten in irgendeiner Hinsicht anders?

Reply:
Bin auch confused. Warum nimmst nicht einfach tidy_finance.sqlite?

Reply:
Es ist keine komplette Kopie, da ich teilweise andere Variablen brauche. Zusätzlich finde ich, dass ein Blog post grundsätzlich nicht in unsere Buch SQL soll, weil das mit externen Authoren nicht einfach wird.

Der Code chunk ist genau dort, damit ich weiß, dass alles aus einer Hand kommt. Den Chunk zu diskutieren wäre aber zu aufwendig.